embeddings/transformation/transformation.py

```diff
@@ -2,13 +2,18 @@
 from typing import TypeVar, Generic
 from abc import ABC
 
+from embeddings.utils.loggers import get_logger
+
 Input = TypeVar("Input")
 Output = TypeVar("Output")
 
 
 class Transformation(ABC, Generic[Input, Output]):
     def __init__(self) -> None:
-        pass
+        self._logger = get_logger(str(self))
```

Comment:
nie wiem czy w pythonie nie ma praktyki ze jest logger per plik a nie instancje?

Reply:
So how it should be resolved?  Logger instance should be passed as parameter and logger management handled by script, that uses it e.g. `document_classification` script? And therefore, should logger be optional?

Reply:
logger was moved into global variable in `embeddings/transformation/flair_transformation/corpus_transformations.py`, it is resolved for me

Reply:
I think that logging should be controlled via some global flag and enabled as default only for scripts.